keps/sig-node/2570-memory-qos/README.md

```diff
@@ -104,6 +107,62 @@ Brief map as follows:
 | pod | sum(requests.memory) | n |
 | node | n/a | pods, kube-reserved, system-reserved | n |
 
+#### Beta
+The formula for memory.high is modified in Beta version of the feature. It will be set based on formula:
+```
+memory.high=floor[(requests.memory + memory throttling factor * (limits.memory or node allocatable memory - requests.memory))/pageSize] * pageSize, where default value of memory throttling factor is set to 0.9
+```
+
+e.g. If a container has `requests.memory=50, limits.memory=100`, and we have a throttling factor of .9, `memory.high` would be 90. If a container had no memory limit specified, we substitute `limits.memory` for `node allocatable memory` and apply the throttling factor of .9 to that value.
+
+Node reserved resources(kube-reserved/system-reserved) are either considered. It is tied to `--enforce-node-allocatable` and `memory.min` will be set properly.
+
+Brief map as follows:
+| type | memory.min | memory.high |
+| -------- | -------- | -------- | 
+| container | requests.memory | floor[(requests.memory + memory throttling factor * (limits.memory or node allocatable memory - requests.memory))/pageSize] * pageSize |
+| pod | sum(requests.memory) | n |
+| node | n/a | pods, kube-reserved, system-reserved | n |
+
+
+**Note:** The formula for memory.high is changed in Beta as the Alpha implementation has following problems:
+1. It fails to throttle when requested memory is closer to memory limits (or node allocatable): memory.high < requests.memory. e.g. If `requests.memory = 85, limits.memory=100`, and we have a throttling factor of 0.8, then as per the Alpha implementation memory.high =  memory throttling factor * limits.memory i.e. memory.high = 80. In this case the level at which throttling is supposed to occur i.e. memory.high is less than requests.memory. Hence there won't be any throttling as the Alpha implementation doesn't allow memory.high to be less than requested memory. 
+
+2. Default throttling factor of 0.8 might be too aggressive for some applications that are latency sensitive and always use memory close to memory limits.
+
+For detailed reasoning around the change, read the documentation [`here`](https://docs.google.com/document/d/1r-e_jWL5EllBgxtaNb9qI2eLOKf9wSuuuglTySWmfmQ/edit?usp=sharing).
```

Comment:
It may be better to introduce the relevant bits here from the document.

Reply:
sure, most of the relevant bits are added except the table. Do you think any other parts of the documentation should be added here?

Reply:
+1, I would copy the relevant info from the doc into the KEP, since the KEP will be the main doc for this feature. I think adding the table might help as well to illustrate the reasoning behind the change.